zio-http/src/main/scala/zio/http/socket/SocketApp.scala

```diff
@@ -20,11 +20,7 @@ import zio._
 
 import zio.http._
 
-final case class SocketApp[-R](
-  decoder: SocketDecoder = SocketDecoder.default,
-  protocol: SocketProtocol = SocketProtocol.default,
-  message: Option[ChannelEvent[WebSocketFrame, WebSocketFrame] => ZIO[R, Throwable, Any]] = None,
-) { self =>
+final case class SocketApp[-R](run: WebSocketChannel => ZIO[R, Throwable, Any]) { self =>
```

Comment:
I think at this point, `SocketApp` is not pulling its weight, and we could juse use:

```scala
type SocketApp[-R] = 
   Handler[-R, Throwable, WebSocketChannel, Any]
```